src/Core/InfoBar/IInfoBarManager.cs

```diff
@@ -54,6 +54,15 @@ public interface IInfoBarManager
         /// </remarks>
         IInfoBar AttachInfoBarWithButtons(Guid toolWindowGuid, string message, IEnumerable<string> buttonTexts, SonarLintImageMoniker imageMoniker);
 
+        /// <summary>
+        /// Attach an info bar with a message and multiple buttons to the main window
+        /// </summary>
+        /// <remarks>
+        /// If there is one buttonText it will be rendered as a button on the info bar.
+        /// If there are multiple buttonTexts they will all be rendered as hyperlinks.
+        /// </remarks>
+        IInfoBar AttachInfoBarToMainWindow(string message, SonarLintImageMoniker imageMoniker, params string[] buttonTexts);
+
```

Comment:
As discussed offline, I removed the two overloads to just leave the one method we need for now.